### ISSUE TITLE
NTR: C18: stearoylcarnitine low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8426]
New term request C18: stearoylcarnitine low in blood (elthat_190106122755):

**Comment by pnrobinson** [comment]
@MickeySegal
I am assuming you meant C18:0

Adding new term: 
- ID: HP:6000437
- label: Reduced criculating stearoylcarnitine concentration
- parents: [Reduced circulating acylcarnitine concentration (HP:0033445)](https://hpo.jax.org/app/browse/term/HP:0033445)
- pmid: 
- def: The concentration of stearoylcarnitine (C18:0) in the blood circulation is below the lower limit of normal.
- comment: Stearoylcarnitine, also known as acylcarnitine C18:0, is a fatty ester lipid molecule.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8426
- synonyms:
